VirtoCommerce.CacheModule.Web/packages.config

```diff
@@ -5,17 +5,17 @@
   <package id="Common.Logging.Core" version="3.3.1" targetFramework="net461" />
   <package id="CommonServiceLocator" version="1.3" targetFramework="net461" />
   <package id="EntityFramework" version="6.2.0" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi" version="5.2.4" targetFramework="net461" />
```

Comment:
You should avoid the modules NuGet dependencies update without a good reason. Due to that fact that the platform can have the same dependencies and them will be taken from platform bin folder and them versions may be different.

Reply:
Last release VirtoCommerce.CoreModule.Web [references](https://github.com/VirtoCommerce/vc-module-core/blob/master/VirtoCommerce.CoreModule.Web/packages.config) to v5.2.4 and without update this module wasn't build.